packages/common/aws.js

```diff
@@ -12,6 +11,28 @@ const { inTestMode, randomString, testAwsClient } = require('./test-utils');
 const promiseRetry = require('promise-retry');
 const pump = require('pump');
 
+/**
+ * Join strings into an S3 key without a leading slash or double slashes
+ *
+ * @param {Array<string>} tokens - the strings to join
+ * @returns {string} the full S3 key
+ */
+function s3Join(tokens) {
```

Comment:
This is a cool utility - I am just curious about why the strings in the `tokens` argument are called `tokens` and not something like `paths` or `pathParts`

Reply:
I guess because I'm really old and used to do C and C++ coding.

http://man7.org/linux/man-pages/man3/strtok.3.html

I can change it to "paths" if you'd like, since that's what `path.join()` uses.